.github/workflows/kitchen-sink.yml

```
@@ -14,6 +14,19 @@ jobs:
       - name: Checkout Code
         uses: actions/checkout@v1
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build image
+        uses: docker/build-push-action@v2
+        with:
+          push: false
+          load: true
+          context: .
+          file: ./Dockerfile
+          tags: |
+            saucelabs/stt-cypress-mocha-node:latest
```

Hm, for the sake of disambiguation it might be more preferable to rename the image tag to `local` and specify that in the config.yml as well. That way, there is no way that we could by accident pick up a pre-built image from docker hub (e.g. if this step were to fail silently for some reason).